static/app/views/settings/organization/organizationPermissionAlert.tsx

```diff
@@ -0,0 +1,36 @@
+import type {ReactNode} from 'react';
+
+import Access from 'sentry/components/acl/access';
+import {Alert, type AlertProps} from 'sentry/components/alert';
+import {t} from 'sentry/locale';
+import type {Scope} from 'sentry/types/core';
+
+export interface OrganizationPermissionAlertProps extends Omit<AlertProps, 'type'> {
+  access?: Scope[];
+  message?: ReactNode;
+}
+
+/**
+ * @deprecated Use `OrganizationPermissionAlert` instead.
+ */
+export function OrganizationPermissionAlert({
```

Comment:
this seems off - I think you wanted to put the `deprecation` warning on `sentry/views/settings/organization/permissionAlert` ?

Reply:
🤦🏼 yes